application/positionmanager/pyproject.toml

```diff
@@ -15,11 +15,12 @@ dependencies = [
     "ecos>=2.0.14",
     # "scipy>=1.12.0",
```

Comment:
Not sure why but these commented bits are back probably just because it's branched from `master` - if this gets merged after the other PRs ignore this comment.

Reply:
removed